go-controller/pkg/ovn/policy_test.go

```diff
@@ -488,11 +489,6 @@ func getHairpinningACLsV4AndPortGroup() []libovsdbtest.TestData {
 	return []libovsdbtest.TestData{egressACL, ingressACL, clusterPortGroup}
 }
 
-func eventuallyExpectNoAddressSets(fakeOvn *FakeOVN, peer knet.NetworkPolicyPeer, namespace string) {
-	dbIDs := getPodSelectorAddrSetDbIDs(getPodSelectorKey(peer.PodSelector, peer.NamespaceSelector, namespace), DefaultNetworkControllerName)
-	fakeOvn.asf.EventuallyExpectNoAddressSet(dbIDs)
-}
-
 func eventuallyExpectAddressSetsWithIP(fakeOvn *FakeOVN, peer knet.NetworkPolicyPeer, namespace, ip string) {
```

Comment:
what about the following two `eventuallyExpectAddressSetsWithIP` and `eventuallyExpectEmptyAddressSetsExist` functions that seem to use fakeOVN.asf, we shouldn't be using these anymore right? given we want to remove fake address set factory...

Reply:
moved these two functions to where they are still used pkg/ovn/pod_selector_address_set_test.go